src/stcal/jump/jump.py

```diff
@@ -609,25 +607,38 @@ def make_snowballs(gdq, integration, group, jump_ellipses, sat_ellipses,
     # center
     # of the saturation circle within the enclosing jump rectangle.
     snowballs = []
+    num_groups = gdq.shape[1]
     for jump in jump_ellipses:
-        if near_edge(jump, low_threshold, high_threshold):
+        # center of jump should be saturated
+        jump_center = jump[0]
+        # if center of the jump ellipse is not saturated in this group and is saturated in
+        # the next group add the jump ellipse to the snowball lisr
+        if group < (num_groups - 1) and \
```

Comment:
Use parentheses, instead of '\' to capture multi-line conditional.

Reply:
fixed